examples/java-matter-controller/java/src/com/matter/controller/commands/discover/DiscoverCommand.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.discover
+
+import chip.devicecontroller.ChipDeviceController
+import com.matter.controller.commands.common.CredentialsIssuer
+import com.matter.controller.commands.common.MatterCommand
+import java.util.concurrent.atomic.AtomicLong
+
+class DiscoverCommand(controller: ChipDeviceController, credsIssuer: CredentialsIssuer?) :
+  MatterCommand(controller, "resolve", credsIssuer) {
+  private val mNodeId: AtomicLong = AtomicLong()
+  private val mFabricId: AtomicLong = AtomicLong()
```

Comment:
The hungarian notation shouldn't be used for Kotlin properties. These can just be:

```
private val nodeId = AtomicLong()
private val fabricId = AtomicLong()
```

Reply:
Acked